cmd/minikube/cmd/ssh.go

```diff
@@ -28,7 +28,7 @@ import (
 
 // sshCmd represents the docker-machine ssh command
 var sshCmd = &cobra.Command{
-	Use:   "ssh",
+	Use:   "ssh [-- COMMAND]",
```

Comment:
hmm, this is one way of doing this, but depending on the command you want to run the '--' might or might not be needed.